index.js

```diff
@@ -0,0 +1,80 @@
+const apiUrl = 'https://ghibliapi.herokuapp.com';
+let localMovieReviewStorage = {};
+
+document.addEventListener('DOMContentLoaded', () => {
+	seedSelectBox();
+	let selectBox = document.querySelector('.movie-titles');
+	selectBox.addEventListener('change', getMovieInfo);
+	let form = document.querySelector('.user-review');
+	form.addEventListener('submit', makeNewReview);
+});
+
+async function seedSelectBox(){
+	let selectBox = document.querySelector('.movie-titles');
+	let movies = await axios.get(`${apiUrl}/films`);
+	let defaultOption = document.createElement('option');
+	selectBox.appendChild(defaultOption);
+	for(let i = 0; i < movies.data.length; i++){
+		let option = document.createElement('option');
+		option.text = movies.data[i].title;
+		localMovieReviewStorage[movies.data[i].title] = [];
```

Comment:
Nice initialization of review storage using [computed properties](https://tylermcginnis.com/computed-property-names/)